workspaces/src/worker/impls.rs

```diff
@@ -144,8 +144,27 @@ where
         )
     }
 
+    /// View account details of a specific account on the network.
+    pub fn view_account(&self, account_id: &AccountId) -> Query<'_, ViewAccount> {
+        Query::new(
+            self.client(),
+            ViewAccount {
+                account_id: account_id.clone(),
+            },
+        )
+    }
+
+    pub fn gas_price(&self) -> Query<'_, GasPrice> {
+        Query::new(self.client(), GasPrice)
+    }
+}
+
+impl<T> Worker<T>
+where
+    T: NetworkClient + Send + Sync + ?Sized,
```

Comment:
What is the motivation here? Why some methods require more constraints than others?

Reply:
Because in this block, the methods are asynchronous and with such bounds, we tell the compiler that type `T` will be moved between threads safely.